README.md

```diff
@@ -93,195 +93,38 @@ Mapbox has a free tier, however there are paid options available. For most users
+2. Clone the git repository: `git clone https://github.com/Zenysis/Harmony`. (Alternatively, you may want to fork the repo and clone the fork — that way you can use version control for your customization.)
+3. Build your development docker images: `docker compose build` (this will take some time!)
 
-### PostgreSQL
-
-You will have to set up a [PostgreSQL database](https://www.postgresql.org/) to host relational data and web application state.
-
-1.  Install postgres.
-
-    - macOS: `brew install postgresql`
-    - Ubuntu: `sudo apt install postgresql`
-
-2.  Start postgres server: `./scripts/db/postgres/dev/start_postgres.sh`
-
-3. [Ubuntu only as this is the default setup on macOS]: The postgresql authentication rules will need to change to allow easier access and the server to connect to postgres. This requires editing the `pg_hba.conf` file and restarting postgres. Here is more information about the configuration file and what changes are being made https://www.postgresql.org/docs/current/auth-pg-hba-conf.html.
-
-   Find the configuration file by running `sudo -u postgres psql -c "SHOW hba_file;"`. Edit the first uncommented line of that file to be
-      ```
-      local   all             postgres                                trust
-      ```
-   Then, add the following lines at the bottom:
-      ```
-      # Allow any user on the local system using local loopback TCP/IP connections
-      # to connect to any database with any database user name.
-      host    all             all             127.0.0.1/32            trust
-      # The same over IPv6.
-      host    all             all             ::1/128                 trust
-      ```
-   Finally, run `sudo service postgresql restart` for the changes to take effect.    ​
-
-4.  Enter psql client to check server success. If it works, then you can exit psql.
-
-    - macOS: `psql postgres`
-    - Ubuntu: `psql -U postgres`
-
-5.  Create and upgrade the local postgres database as well as populate the Data Catalog tables: `./scripts/db/postgres/dev/init_db.py --populate_indicators`
-
-    This command will create a database for each deployment defined in config, you can optionally run the command for just one deployment like `./scripts/db/postgres/dev/init_db.py <ZEN_ENV> --populate_indicators`. Also you can run just `./scripts/db/postgres/dev/init_db.py` to only upgrade the database and not populate Data Catalog.
-
-6. Create an admin user for your local web app like below.
-
-      ```
-      ./scripts/create_user.py -f <first name> -l <last name> -u <email> -p <password> -a -d postgresql://postgres:@localhost/{ZEN_ENV}-local
-      ```
-
-### Hasura
-
-Start Hasura: `./scripts/db/hasura/dev/start_hasura.sh <ZEN_ENV>-local`
+### Run Web Server
 
-### Run webpack locally
+1. Start your development environment: `DEFAULT_DRUID_HOST=<public production Druid server IP> ZEN_ENV=<specify environment> docker compose up` (you need to specify druid host and environment)
```

Comment:
I tried this with `DEFAULT_DRUID_HOST=aws-druid.corp.clambda.com ZEN_ENV=br docker compose up` and it failed with the below error:

```
harmony-web-1       | Traceback (most recent call last):
harmony-web-1       |   File "/app/web/runserver.py", line 97, in <module>
harmony-web-1       |     sys.exit(main())
harmony-web-1       |   File "/app/web/runserver.py", line 92, in main
harmony-web-1       |     app = create_app(flask_config, instance_config, environment)
harmony-web-1       |   File "/app/web/server/app.py", line 455, in create_app
harmony-web-1       |     app = _create_app_internal(flask_config, instance_config, configuration_module)
harmony-web-1       |   File "/app/web/server/app.py", line 186, in _create_app_internal
harmony-web-1       |     _initialize_druid_context(app)
harmony-web-1       |   File "/app/web/server/app.py", line 253, in _initialize_druid_context
harmony-web-1       |     if datasource_config in druid_metadata.get_datasources_for_site(deployment_name):
harmony-web-1       |   File "/app/db/druid/metadata.py", line 98, in get_datasources_for_site
harmony-web-1       |     return [ds.name for ds in self.get_site_datasources() if ds.site == site]
harmony-web-1       |   File "/app/db/druid/metadata.py", line 91, in get_site_datasources
harmony-web-1       |     for ds in self.get_all_datasources()
harmony-web-1       |   File "/app/db/druid/metadata.py", line 81, in get_all_datasources
harmony-web-1       |     r = requests.get(url)
harmony-web-1       |   File "/app/venv/lib/python3.9/site-packages/requests/api.py", line 73, in get
harmony-web-1       |     return request("get", url, params=params, **kwargs)
harmony-web-1       |   File "/app/venv/lib/python3.9/site-packages/requests/api.py", line 59, in request
harmony-web-1       |     return session.request(method=method, url=url, **kwargs)
harmony-web-1       |   File "/app/venv/lib/python3.9/site-packages/requests/sessions.py", line 587, in request
harmony-web-1       |     resp = self.send(prep, **send_kwargs)
harmony-web-1       |   File "/app/venv/lib/python3.9/site-packages/requests/sessions.py", line 695, in send
harmony-web-1       |     adapter = self.get_adapter(url=request.url)
harmony-web-1       |   File "/app/venv/lib/python3.9/site-packages/requests/sessions.py", line 792, in get_adapter
harmony-web-1       |     raise InvalidSchema(f"No connection adapters were found for {url!r}")
harmony-web-1       | requests.exceptions.InvalidSchema: No connection adapters were found for 'aws-druid.corp.clambda.com:8081/druid/coordinator/v1/metadata/datasources'
harmony-web-1       | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
harmony-web-1       | error Command failed with exit code 1.
harmony-web-1 exited with code 1
```

Reply:
Missing http:// - needs to be `DEFAULT_DRUID_HOST=http://aws-druid.corp.clambda.com ZEN_ENV=br docker compose up` - I've updated the readme with an example.